ocs_ci/ocs/ui/validation_ui.py

```diff
@@ -179,7 +179,8 @@ def odf_console_plugin_check(self):
         """
         self.ocp_version = get_ocp_version()
         ocs_version = version.get_semantic_ocs_version_from_config()
-        if self.ocp_version >= "4.9" and ocs_version >= version.VERSION_4_9:
+        ocp_version = version.get_semantic_ocp_version_from_config()
+        if ocp_version >= version.VERSION_4_9 and ocs_version >= version.VERSION_4_9:
```

Comment:
You can use `self.ocp_version_semantic`
https://github.com/red-hat-storage/ocs-ci/blob/447fc6c4eb54ba038fc613eb8a1df32577140d5a/ocs_ci/ocs/ui/base_ui.py#L342

Reply:
Is the current implementation incorrect?

Reply:
I think it's exactly the same except for variable name

Reply:
We have var in `Page_Nav` class. so we can use it..[OOP inheritance]
https://www.geeksforgeeks.org/inheritance-in-python/